src/components/HeaderInfo/HeaderInfo.js

```diff
@@ -227,26 +227,26 @@ export const HeaderInfo = ({
     workspaceSection === MONITORING_PLAN_STORE_NAME
       ? "Monitoring Plan"
       : workspaceSection === EMISSIONS_STORE_NAME
-      ? "Emissions"
-      : "Test";
+        ? "Emissions"
+        : "Test";
 
   const MAX_REPORTING_PERIODS = 4;
   const MAX_REPORTING_PERIODS_ERROR_MSG =
     "You can only select a maximum of four reporting periods";
 
-  let reportingPeriods = useMemo(
+  const reportingPeriods = useMemo(
```

Comment:
Can you lave this as let instead of const. This variables looks like it gets reassigned else where in the code from what I'm seeing inside the handleSelectedReportingPeriod function. If it is const then it'll throw an error when that function runs.

Reply:
Reverted to const.